src/ElementFormeEtoile.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Ipssi\Evaluation;
+
+class ElementFormeEtoile extends IPSSI\Evaluation\ElementForme {
```

Comment:
```suggestion
class Etoile extends IPSSI\Evaluation\ElementForme {
```
C'est plus simple :)